core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -161,7 +161,7 @@ object Controller {
   // a value, and whose values are default values.   A null value in the Map means there is
   // no default value specified, so it must appear in the properties file
   def requiredProperties =
-    Map(WhiskConfig.controllerInstances -> null) ++
+    Map(WhiskConfig.controllerInstances -> null, WhiskConfig.controllerProtocol -> null) ++
```

Comment:
Pure config as well.

But if you really need to keep it in WhiskConfig, it is already in `RestApiCommons.requiredProperties `. So you don't need to add it here.